src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -754,188 +747,163 @@ async def _genomic_to_tx_segment(
             following transcripts: MANE Select, MANE Clinical Plus, Longest Remaining
             Compatible Transcript
         :param gene: Valid, case-sensitive HGNC gene symbol
-        :param get_nearest_transcript_junction: If ``True``, this will return the
-            adjacent exon if the position specified by``seg_start_genomic`` or
-            ``seg_end_genomic`` does not occur on an exon. For the positive strand, adjacent
-            is defined as the exon preceding the breakpoint for the 5' end and the exon
-            following the breakpoint for the 3' end. For the negative strand, adjacent
-            is defined as the exon following the breakpoint for the 5' end and the exon
-            preceding the breakpoint for the 3' end.
         :param is_seg_start: ``True`` if ``genomic_pos`` is where the transcript segment starts.
             ``False`` if ``genomic_pos`` is where the transcript segment ends.
+        :param coordinate_type: Coordinate type for ``seg_start_genomic`` and
+            ``seg_end_genomic``. Expects inter-residue coordinates by default
         :return: Data for a transcript segment boundary (inter-residue coordinates)
         """
         params = {key: None for key in GenomicTxSeg.model_fields}
 
-        if get_nearest_transcript_junction:
-            if not gene and not transcript:
-                return GenomicTxSeg(
-                    errors=[
-                        "`gene` or `transcript` must be provided to select the adjacent transcript junction"
-                    ]
-                )
-
-            if not genomic_ac:
-                genomic_acs, err_msg = self.seqrepo_access.chromosome_to_acs(chromosome)
+        if not gene and not transcript:
+            return GenomicTxSeg(errors=["`gene` or `transcript` must be provided"])
```

Comment:
It'd be nice to have better error handling at some point

Reply:
I agree. Is there a similar way we handle errors in our other apps?

Reply:
We can address in #161